packages/react-native-skia/components/text/NumberOfLines_ellipSizeMode.js

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+import { AppRegistry, Text, View } from 'react-native';
```

Comment:
View is not used , so probably not required to import.

Reply:
Yes, removed

Reply:
Updated, Removed View